products/compute/terraform.yaml

```diff
@@ -256,17 +256,19 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         name: backend
         is_set: true
         set_hash_func: 'resourceGoogleComputeBackendServiceBackendHash'
+      backends.failover: !ruby/object:Overrides::Terraform::PropertyOverride
+        min_version: beta
```

Comment:
Can we add this on the descriptions in api.yaml instead? Then every provider gets them